docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -46,6 +46,86 @@ apb init
 
 NOTE: Best practice is to name the APB ending with `-apb`.
 
+== Making your APB consumable as a mobile service
+
+When creating an APB for a mobile service, where possible use Kubernetes resources
+( rather than Openshift resources ) to ensure compatibility with Openshift and Kubernetes.
+
+Only one instance of a service is expected in a namespace. If your service is intended to have more
+than one service instance consideration must be given to the naming of objects.
+
+=== ConfigMaps Vs Secrets
+
+Separate the public and private configuration information in your APB.
+
+Use *configMaps* for public information that a mobile client will need to use the service and label the configMap
+with mobile: enabled and the serviceName so it can be read by the mobile-cli.
+
+An example of this can be seen link:https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/provision-keycloak-apb/templates/configmap.yml.j2[here].
+
+[source,yaml]
+----
+  labels:
+    mobile: enabled
+    serviceName: my-custom-service
```

Comment:
It would be good to call out here that some 'secret' info may be required here anyways. e.g. an API Key that the mobile client has to use.

Reply:
@josemigallas i think it's an example for both ConfigMap and Secret i.e. add these labels to both, if you have both

Reply:
@josemigallas Yeah it's an example specifically from a configMap but I think it works for either

Reply:
Right under this I have an example secret which has the same labels.

Reply:
I just wonder if it would be possible to give this example a little bit more of flavour, like the secret one below. I as a developer would like to simply copy and paste the whole snippet in order to add a config map in my APB. I mean, for consistency this snippet should be an example of a config map, not both, since it belongs to the paragraphs where config maps are described. I would rather remove the "an example here.." link and extend the snippet. IMO it would be a bit clearer this way.

Reply:
Good point, i'll make that clearer